### PR TITLE
remove makefile:generated_files in v1.29.

### PR DESCRIPTION
**Pull request by taoyounger**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it:**

**Which issue(s) this PR fixes:**
Fixes #

**Special notes for your reviewer:**
Does this PR introduce a user-facing change?
```
TODO(thockin): Remove this in v1.29.
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:**

### CODE REVIEW

**Comment by xmudrii** [comment]
cc @thockin as he left that TODO

**Comment by thockin** [comment]
Thanks!  Obviously needs to wait until master branch opens again.

/lgtm
/approve

File: build/root/Makefile [hunk 1]
```diff
@@ -311,16 +311,6 @@ clean:
 	hack/make-rules/clean.sh
 endif
 
-
-# TODO(thockin): Remove this in v1.29.
-.PHONY: generated_files
-generated_files:
-	echo "'make generated_files' is deprecated.  Please use hack/update-codegen.sh instead."
```
Comment (chendave) [inline build/root/Makefile:318]: well, this target is removed, is there anywhere in doc to introduce where to find the script/how to generate the code?

**Pull request merged**